### DISCUSSION TITLE
Audio cuts, server freezes

### DISCUSSION THREAD

**Discussion by den0rk** [apohl79/audiogridder#338] [Help]
I have resumed trying AudrioGridder again and am having problems ..

I am using a direct PC to PC connection with 1GB crossover cable.

The first time I tried AudioGridder it was perfect, I opened 10 AA Cerise without problem. When I have closed and opened the project the problems have already started, I do not know what can happen, the AudioGridder server is practically frozen all the time, that generates audio cuts in the host, the server CPU.

I remove 5 Cerise, it seems that now it works, I save, close and open the project, the problem returns.

The monitor fX plugin shows red dots randomly in the different instances.

the server CPU is not even 40%, sometimes when there is no clipping then the audio is out of sync.
[AudioGridderPlugin_2021-01-29_23-59-12.log](https://github.com/apohl79/audiogridder/files/5896799/AudioGridderPlugin_2021-01-29_23-59-12.log)

**Comment by apohl79** [comment]
As I stated before, more info is needed, as this sounds like server issues. Thx.

**Reply by den0rk** [reply]
[AudioGridderServer_2021-01-30_00-07-45.log](https://github.com/apohl79/audiogridder/files/5896844/AudioGridderServer_2021-01-30_00-07-45.log)

**Reply by apohl79** [reply]
Looks like the two logs are not matching the same test run. Please start a clean session on the server an client/DAW side and provide the logs. Also please activate tracing on the server and plugin side. I don't see issues in the server log.

When you say the server is frozen, how does it recover?

Can you test your network latency and throughput? With the crossover cable I would not expect anything there, but the server reports an audio processing performance of ~4ms (95th) and the plugin ~6ms. The problem is the two logs don't match, but 2ms sounds pretty large on the network side, given your direct connection.

**Reply by den0rk** [reply]
Today I opened the same project and it worked fine, when I added Lemon, it started to crash.

I attach all the logs. https://we.tl/t-5pWy9W8zJE

The network between the teams is good, this is the result of the test.

![image](https://user-images.githubusercontent.com/66702706/106353453-f5c30480-62ea-11eb-9e28-9d85e22c28a3.png)

**Reply by den0rk** [reply]
The server does not recover, even when I close the DAW, the server stays frozen most of the time.

**Reply by apohl79** [reply]
Ok, to help you further, you have to follow the bug reporting steps: https://audiogridder.com/bug-reports/

Please provide server and plugin logs of one test run. Activate tracing etc. Also if the server hangs, there is a guide to create a core dump.

**Reply by den0rk** [reply]
in the wetransfer link I attached a dmp,

**Reply by den0rk** [reply]
Hi, the problem may be solved, I'll keep trying to confirm.

I use a Merging Anubis audio card and it works over the network.

I have a network for it, I had it with a fixed IP, now I have configured it with DHCP.

Maybe the problem was off the network, the system did not know where to send the traffic.

I confirm if it has been resolved when I do more tests.

**Reply by den0rk** [reply]
Andreas, it is working fine now!

Sorry for the inconvenience as it was a problem with my network.

**Reply by apohl79** [reply]
happy days! glad it works now!

**Comment by CaBuffalo** [comment]
Try change the buffer size to different blocks and see if it helps.

On Fri, Jan 29, 2021 at 3:11 PM Andreas Pohl <notifications@github.com>
wrote:

> As I stated before, more info is needed, as this sounds like server
> issues. Thx.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/338#discussioncomment-322424>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HIQZX7KFXQVRLFWOLS4M6A7ANCNFSM4WZVQNCQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by den0rk** [reply]
Tried several, it doesn't help.

**Comment by CaBuffalo** [comment]
I understand that Acustica Plugins are tricky sometimes. Did you try
different EQs?

I'm using all different kinds of plugins. So far, it runs stable to me.

On Fri, Jan 29, 2021 at 3:20 PM den0rk <notifications@github.com> wrote:

> Tried several, it doesn't help.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/338#discussioncomment-322443>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FUH35XJCXPGJOSM6LS4M7ENANCNFSM4WZVQNCQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
